src/pages/recipes/[recipeId].tsx

```diff
@@ -54,27 +45,21 @@ export default function RecipeDetail() {
         <DataTable.Skeleton columns={4} rows={18} />
```

Comment:
can we refactor this one also to have the same component pattern with the recipes page?

Reply:
sorry for the iterative comments, this is not related to the PR but since we're cleaning up the code... :)

Reply:
sure

Reply:
fixed